src/components/include/protocol_handler/session_observer.h

```diff
@@ -178,6 +178,13 @@ class SessionObserver {
    */
   virtual void OnMalformedMessageCallback(const uint32_t& connection_key) = 0;
 
+  /**
+   * @brief Callback function used by ProtocolHandler when last messages were
+   * sent for the connection
+   * @param connection_key used by other components as application identifier
```

Comment:
```suggestion
   * @param connection_key used by other components as an application identifier
```

Reply:
@IGapchuk 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/136/commits/957e39c33032718558882f177d2cb17a44624afc